### PR TITLE
WM3 - Abiya Yaccobu - JavaScript1 - Week 2

### PR DESCRIPTION
**Pull request by AbiyaY**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: extra/1-radio-stations.js [hunk 1]
```diff
@@ -25,7 +33,17 @@
...
+  let allFrequency = getAllFrequencies();
+  let availableFrequency = [];
+  for(let i = 0; i < allFrequency.length; i++) {
+    let frequencies = allFrequency[i];
+    if(isRadioStation(frequencies)) {
+      availableFrequency.push(frequencies);
+    }
+  }
+  return availableFrequency;
+}
```
Comment (Haseeb3927) [inline extra/1-radio-stations.js:46]: Well done Abiyaa good work. I'm impressed the way you did the extra task

Comment (AbiyaY) [reply extra/1-radio-stations.js:46]: Thanks, Haseeb. I got some useful suggestions from HWC.

**Comment by AbiyaY** [comment]
> Clean and straight forward code....well done!



> Clean and straight forward code....well done!

Thanks Georgina!

**Comment by AbiyaY** [comment]
> Clean and straight forward code....well done!

Thanks Georgina!

**Pull request closed without merging**